src/task1.js

```diff
@@ -0,0 +1,5 @@
+(() => {
```

Comment:
Consider removing this IIFE since there's no need to have a local scope in Node - its files already follow a module pattern and have local scope by default.